src/main/java/br/com/zup/concessionariacitroen/services/VendaService.java

```diff
@@ -0,0 +1,42 @@
+package br.com.zup.concessionariacitroen.services;
+
+import br.com.zup.concessionariacitroen.exceptions.NaoHaVendasCadastradasException;
+import br.com.zup.concessionariacitroen.models.Venda;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class VendaService {
+
+    private static List<Venda> listaVendas = new ArrayList<>();
+
+    public Venda efetuarVenda(Venda venda){
+        listaVendas.add(venda);
+        return venda;
+    }
+
+    public List<Venda> listarVendas(){
+        return listaVendas;
+    }
+
+    public List <Venda> pesquisarVendaPeloCPFdoCliente(String cpf) {
+        List<Venda> vendasDoClienteInformado = new ArrayList<>();
+        verificarListaVendasPorClienteEstaVazia();
+        for (Venda objetoVenda : listaVendas){
+            if (objetoVenda.getNumeroPedido().getCliente()
+                    .getCpf().equals(cpf)) {
+
+                vendasDoClienteInformado.add(objetoVenda);
+            }
+        }
+        return vendasDoClienteInformado;
+    }
+
+    private void verificarListaVendasPorClienteEstaVazia(){
```

Comment:
Mudar o nome do metodo.
```suggestion
    private void verificarSeHaVendas(){
```